src/contracts/pods/EigenPod.sol

```diff
@@ -164,6 +164,7 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
          * initialized with hasRestaked set to true.
          */
         hasRestaked = true;
+        emit RestakingActivated(podOwner);
```

Comment:
IIRC we set `hasRestaked = true` in other places as well!

Reply:
are those just already covered then?

Reply:
ye we set it at one other place, and we are only emitting there before this change :)